pkg/apis/v1alpha1/subnet_types.go

```diff
@@ -30,9 +30,11 @@ type SubnetSpec struct {
 
 // SubnetStatus defines the observed state of Subnet.
 type SubnetStatus struct {
-	NSXResourcePath string      `json:"nsxResourcePath,omitempty"`
-	IPAddresses     []string    `json:"ipAddresses,omitempty"`
-	Conditions      []Condition `json:"conditions,omitempty"`
+	NSXResourcePath   string      `json:"nsxResourcePath,omitempty"`
+	NetworkAddress    []string    `json:"networkAddress,omitempty"`
```

Comment:
So, in NSX API, these fields is a single string or a string array?

In K8s API naming convention, they should be in plural ("XYZAddresses") if there can be multiple addresses?

Reply:
Yes, in NSX API, these fields are a single string not a string array.So, it's good to keep it as string instead of array to align with NSX API?

Reply:
Then we should use string, rather than []string.

The content of a string is one address or can be multiple addresses? If latter, we should name the fields as "xyzAddresses".

Reply:
Sorry, my bad, NSX API return status list which includes multiple NetworkAddress. We should name this field to NetworkAddresses as the plural type.